### PR TITLE
LPS-121302 - Remove no needed aria-expanded

### PR DESCRIPTION
**Pull request by marcoscv-work**
Remove no needed aria in order to fix validation error:

image

--- 

**How to test this:**

- Go to a widget page.
- Get the main navigation markup
- Use validator.w3.org to validate it

### CODE REVIEW

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

File: modules/apps/frontend-theme/frontend-theme-classic/src/templates/navigation.ftl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		<span class="navbar-toggler-icon"></span>
 	</button>
 
-	<div aria-expanded="false" class="collapse navbar-collapse" id="navigationCollapse">
```
Comment (jbalsas) [inline modules/apps/frontend-theme/frontend-theme-classic/src/templates/navigation.ftl:6]: Isn't this a collapsible menu on small screen sizes?

Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-classic/src/templates/navigation.ftl:6]: Hh, yeah, but the element which needs the aria-expanded is the trigger, right now we have duplication in the trigger and in the container:

https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-classic/src/templates/navigation.ftl#L2

The relation between trigger an element hidden is done by the `aria-controls="ID"`

An example in bootstrap here:
https://getbootstrap.com/docs/4.5/components/navbar/#supported-content

Comment (jbalsas) [reply modules/apps/frontend-theme/frontend-theme-classic/src/templates/navigation.ftl:6]: fair enough, I just checked the diff here 👍

**Comment by pat270** [comment]
This LGTM!

ci:forward

**Comment by marcoscv-work** [comment]
ci:forward

**Pull request closed without merging**